frontend/src/features/videos/VideoCard.tsx

```diff
@@ -308,17 +338,28 @@ function VideoCard({
                 display="flex"
                 alignItems="center"
                 data-testid="video-card-overall-score"
+                {...(unsafe == true && {
+                  sx: {
+                    filter: 'grayscale(100%)',
+                    opacity: 0.6,
+                  },
+                })}
               >
-                <img
-                  className="tournesol"
-                  src={'/svg/tournesol.svg'}
-                  alt="logo"
-                  title="Overall score"
-                  width={32}
-                />
-                <span className={classes.nb_tournesol}>
-                  {total_score.toFixed(0)}
-                </span>
+                <SafeTournesolScoreWrapper
```

Comment:
Using this wrapper inside the <Box display="flex"> breaks the alignment of the content, as the wrapper consists of a single child.  
I think that it would make sense to use the wrapper above the `<Box>`.